frontend/src/components/ActivityReportsTable/ReportRow.js

```diff
@@ -23,7 +23,7 @@ function ReportRow({
     displayId,
     activityRecipients,
     startDate,
-    topics,
+    sortedTopics: topics,
     lastSaved,
```

Comment:
turns out we had a sorted topics virtual field, who knew

Reply:
Should we be overwriting the sorted topics?